packages/scripts/utils/config.js

```diff
@@ -108,7 +112,9 @@ const hasPostCSSConfig = () =>
  */
 const getWebpackArgs = () => {
 	// Gets all args from CLI without those prefixed with `--webpack`.
-	let webpackArgs = getArgsFromCLI( [ '--webpack' ] );
+	let webpackArgs = getArgsFromCLI( [ '--webpack' ] ).filter(
```

Comment:
Would the following work, too?
```suggestion
	let webpackArgs = getArgsFromCLI( [ '--webpack', '--experimental-modules' ] );
```

Reply:
That works. I avoided it because `getArgsFromCLI` takes an array of "excluded prefixes" and that's not really a prefix. Under the hood it's excluding strings by `startWith` so it should be fine.